target/linux/ramips/image/mt7620.mk

```diff
@@ -924,14 +915,16 @@ define Device/ralink_mt7620a-v22sg-evb
 endef
 TARGET_DEVICES += ralink_mt7620a-v22sg-evb
 
-define Device/ravpower_wd03
+define Device/ravpower_rp-wd03
+  $(Device/sunvalley_loader_okli)
   SOC := mt7620n
-  IMAGE_SIZE := 7872k
-  DEVICE_VENDOR := Ravpower
-  DEVICE_MODEL := WD03
-  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
+  IMAGE_SIZE := 6144k
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-ralink
+  DEVICE_VENDOR := RAVPower
+  DEVICE_MODEL := RP-WD03
+  LZMA_TEXT_START := 0x81800000
```

Comment:
if LZMA_TEXT_START applies to both devices it should be in the common section

However I thought this was set recently as the default in master...so maybe just delete the line?

Reply:
it was linked in forum by @xabolcs 

https://github.com/openwrt/openwrt/commit/a40ddc2195350149aadbdb6077b17d4787becd9d

Reply:
Ummm ... so leave this then? Thanks!

Reply:
nah, its definitely the same value now as the default, so remove line

Reply:
NP, done!